tools/gadget3_to_eagleHDF.py

```diff
@@ -0,0 +1,115 @@
+import numpy as np
+import os
+import argparse
+from scipy.io import FortranFile
+import h5py as h5
+
+"""
```

Comment:
It is common to put the docstring at the very beginning of the script file rather than after the imports.